locale/templates/settings.ftl

```diff
@@ -535,7 +545,7 @@ se-content-note = Note: a secondary email won’t restore your information — y
 tfa-row-header = Two-step authentication
 tfa-row-disabled = Two-step authentication disabled.
 tfa-row-enabled = Enabled
-tfa-row-not-set = Not set
+tfa-row-not-set = Not Set
```

Comment:
Why is this capitalized? The change won't impact l10n if the string is localized.